common.js

```diff
@@ -149,7 +149,11 @@ module.exports = {
       return Promise.resolve()
     }
 
-    return Promise.all(hooks.map(hookFn => pify(hookFn).apply(this, args)))
+    const promisified = hooks.map(hookFn => pify(hookFn).bind(this, ...args))
```

Comment:
What's the right way to do this without the spread operator?

Reply:
In fact it looks like Node 4 will be EOL at the end of April, so I guess this _could_ stay in **if** this gets accepted and released after that